crates/hc_sandbox/src/config.rs

```diff
@@ -2,17 +2,20 @@
 
 use holochain_conductor_api::conductor::paths::ConfigFilePath;
 use holochain_conductor_api::conductor::paths::ConfigRootPath;
+use holochain_conductor_api::conductor::DpkiConfig;
 use holochain_conductor_api::config::conductor::ConductorConfig;
 use holochain_conductor_api::config::conductor::KeystoreConfig;
 
 /// Create a new default [`ConductorConfig`] with data_root_path path,
 /// keystore, and database all in the same directory.
+/// DPKI is disabled.
```

Comment:
Why is DPKI disabled by default for sandboxes?

Reply:
In my view (and I could be wrong since I haven't used one in years) is the sandboxes are meant to be relatively disposable test environments. They have limited functionality anyway, since you have to interact with them through the commandline, so it doesn't seem surprising that you wouldn't be able to update your agent key in them.

Reply:
It's what Tryorama uses, and Tryorama is used by scaffolding for generating all tests. So after all it's probably used the most of all holochain conductors for testing. It's what happ devs use all the time while developing an app. Hence I think DPKI should be enabled by default.

Reply:
I hear you on wanting sandboxes to be feature complete, but they are far from it. For instance, from a quick glance at the [sandbox InstallApp](https://github.com/holochain/holochain/blob/bd7dafb8a5e1f56707dec834572df478a73a67d7/crates/hc_sandbox/src/calls.rs#L161-L179), I see that sandboxes can't accept memproofs nor can they specify UseExisting. I'm sure there are other things sandboxes can't do.

Not that this alone is a justifcation for making them even further from feature complete. But to properly enable DPKI on sandboxes now, you would also need to be able to pass in a device seed to the keystore used by the sandbox. Sandboxes are currently configured to use the DangerTestKeystore, which uses an in-process lair instance and provides no external access, so we'd also have to add in a mechanism for interacting with that keystore. Just seems like a lot of trouble to add a feature to a setup that's already feature incomplete, especially when that feature is used mainly for long-lived networks.

I do see that it's important for devs to test that their apps work well with key updates, so they should have some scenarios involving DPKI. But maybe we cross that bridge when we actually have key updates? The key update working group is still under way, after all.

If tryorama tests are supposed to use a feature-complete holochain, then we either need to dedicate resources to getting sandbox up to date in all the ways, or tryorama should be refactored to just use the normal holochain interface.

Reply:
Not necessarily feature completeness, but I thought if we have the option of providing DPKI to the sandbox conductor, then let's enable it by default. I understand that it's not possible right now without further modification, so let's go ahead as you suggest.